app/models/event.rb

```diff
@@ -73,20 +72,30 @@ def to_s
     name
   end
 
+  def draft?
+    state == 'draft'
+  end
+
   def open?
     state == 'open' && (closes_at.nil? || closes_at > Time.current)
   end
 
   def closed?
-    !open?
+    !open? && !draft?
   end
 
   def past_open?
     state == 'open' && closes_at < Time.current
   end
 
   def status
-    open? ? 'open' : 'closed'
+    if open?
+      'open'
+    elsif draft?
+      'draft'
+    else
+    'closed'
+    end
```

Comment:
I wonder if the strings `open`, `closed` and `draft` can be made into constants at the top of this class to prevent the duplication and potential typos in strings.